mysql-test/suite/innodb/t/percona_changed_page_bmp_shutdown_thread.test

```diff
@@ -0,0 +1,37 @@
+#
+# PS-5642 - page tracker thread won't exist if startup fails
+#
+
+--source include/have_debug.inc
```

Comment:
Can you make this a non-debug test by poking a few bytes into actual redo log header while server is shutdown?

Reply:
@laurynas-biveinis you mean replace some bytes at some specific position? such as last checkpoint on redo log header?
What I'm finding hard here is that page_tracking only starts after the redo log has been initialized. Even though I mess with redo log it will fail on a point where page tracking threads are not started yet.

Reply:
Right. Perhaps it's possible to disrupt InnoDB startup at a point after redo initialization, maybe by deleting data file or setting some later incompatible startup option. If such alternative does not come up with little effort, then OK to leave as a debug test

Reply:
Hi @laurynas-biveinis , I've not been able to come up with a non-debug test that did not affect innodb start-up.

Reply:
OK then